### PR TITLE
Update mumbai tables, nits

### PR DESCRIPTION
**Pull request by sanderpick**

### CODE REVIEW

File: animation_url/src/routes/[slug]/+page.svelte [hunk 1]
```diff
@@ -16,6 +16,7 @@
   </div>
   {#if data.pilot}
     <img class="pilot" src={data.pilot} alt="pilot" />
+    <span class="warning">Status: In-flight (not transferable).</span>
```
Comment (sanderpick) [inline animation_url/src/routes/[slug]/+page.svelte:19]: Adding a warning here about transferability.

File: ethereum/deployments.ts [hunk 2]
```diff
@@ -51,10 +51,10 @@ export const deployments: RigsDeployments = {
...
     contractTable: "rigs_contract_80001_3513",
     allowlistTable: "rigs_allowlist_80001_3514",
-    partsTable: "",
-    layersTable: "",
-    attributesTable: "rig_attributes_80001_3507",
-    lookupsTable: "lookups_80001_3508",
+    partsTable: "parts_42161_2",
+    layersTable: "layers_42161_3",
+    attributesTable: "rig_attributes_42161_4",
+    lookupsTable: "lookups_42161_5",
```
Comment (sanderpick) [inline ethereum/deployments.ts:57]: testing the tables on arbitrum mainnet before deploying all the data there.

File: garage/src/hooks/useRigStats.ts [hunk 3]
```diff
@@ -34,8 +34,8 @@ export const useStats = (currentBlockNumber: number | undefined) => {
         { name: "Num. pilots", value: numPilots },
         { name: "Average FT per flight", value: ftAvg },
         { name: "Total FT earned", value: ftTotal },
-        { name: "Badges earned", value: 0 },
-        { name: "Badges visible", value: 0 },
+        // { name: "Badges earned", value: 0 },
+        // { name: "Badges visible", value: 0 },
```
Comment (sanderpick) [inline garage/src/hooks/useRigStats.ts:38]: hiding these badge stats

**Review by datadanne — APPROVED** [review]
Garage changes lgtm!

**Pull request merged**